in.go

```diff
@@ -99,6 +99,20 @@ func Get(request GetRequest, github Github, git Git, outputDir string) (*GetResp
 		if err := ioutil.WriteFile(filepath.Join(path, filename), content, 0644); err != nil {
 			return nil, fmt.Errorf("failed to write metadata file %s: %s", filename, err)
 		}
+
+	}
+
+	if len(pull.ChangedFiles) != 0 {
```

Comment:
Ref my comment about the function signature of `GetPullRequest`. I think it would make sense if this if statement checked `if request.Params.ChangedFilesQuery` (to be renamed) and listed the changed files right in the if statement. Doesn't even need to add it to the PR object since it's not used anywhere else yet. Thoughts?

Reply:
I moved it around. This actually also made the test code slightly smaller, however conceptually it would have fit into the PR object quite well